tasks/lzards-backup/src/index.ts

```diff
@@ -25,16 +26,24 @@ import {
   ChecksumError,
   CollectionNotDefinedError,
   CollectionInvalidRegexpError,
+  CollectionIdentifiersNotProvidedError,
   GetAuthTokenError,
   InvalidUrlTypeError,
 } from './errors';
 import { isFulfilledPromise } from './typeGuards';
-import { MakeBackupFileRequestResult, HandlerEvent, MessageGranule, MessageGranuleFilesObject } from './types';
+import {
+  MakeBackupFileRequestResult,
+  HandlerEvent,
+  MessageGranule,
+  MessageGranuleFilesObject,
+  MessageGranuleFromStepOutput,
+  ApiGranule,
+} from './types';
 
 const log = new Logger({ sender: '@cumulus/lzards-backup' });
 
 const CREDS_EXPIRY_SECONDS = 1000;
-const S3_LINK_EXPIRY_SECONDS_DEFAULT = 3600;
+const S3_LINK_EXPIRY_SECONDS_DEFAULT = '3600';
```

Comment:
Hmm it wasn't though, the `Expires` property expects a string or number so it should've worked as a number but I was seeing an error 🤔 I'll look into this because perhaps this change is unnecessary